PID Data Model alignment with ARF PID Rulebook v1.3 and CIR 2024/2977 (#913)
italia/eidas-it-wallet-docs: docs/en/credential-data-model-pid.rst

.. include:: ../common/common_definitions.rst

PID Data Model
==============================

> The Person Identification Data (PID) is issued by the PID Provider according to national laws and it MUST be provided in SD-JWT-VC and mdoc-CBOR data format. 

Reviewer: ```suggestion
The Person Identification Data (PID) is issued by the PID Provider according to national laws and it MUST be provided in both SD-JWT VC and mdoc-CBOR data format. 
```

.. note::
   **Transitional Phase:**

>    During the transitional phase before full EUDIW operability,PID will be provided only in SD-JWT-VC format. 

Reviewer: ```suggestion
   During the transitional phase before full EUDIW operability, PID will be provided only in SD-JWT VC format. 
```

- Current Family Name
- Current First Name
- Date of Birth
- Place of Birth
- Nationality
- User identification number across public Relying Party services (for example the *taxpayer identification number*)

In addition to the User attributes listed above, the PID includes also the following metadata attributes (`EU_2024/2977`_ and **Section 2 of the ARF PID Rulebook v1.3** [`EIDAS-ARF`_]):

- Issuing authority
- Issuing country
- Expiry Date
- Validity status information
- Identity and data proofing information 

Some attributes, such as the *taxpayer identification number* and the *identity and data proofing information*, are provided as **domestic extensions** defined by the Italian IT-Wallet specification. It is NOT part of the ARF PID Rulebook (Annex 3.01, PID Rulebook v1.3), but is **permitted under ARF requirement PID_06**, which allows Member States to define additional domestic attributes beyond those specified in Commission Implementing Regulation (CIR) 2024/2977 (`EU_2024/2977`_). In particular, the identity proofing information is REQUIRED for Italian PIDs to ensure:

> - Traceability of User authentication method.

Reviewer: ```suggestion
- The evaluation of User authentication method used.
```

- Level of Assurance compliance of identity proofing during the enrollment process (LoA as defined by eIDAS Regulation).

> - Auditability of identity and User attributes verification processes.

Reviewer: ```suggestion
- The level of Assurance compliance of identity proofing during the enrollment process, according to the LoA defined by the eIDAS Regulation.
- The auditability upon the User attributes verification processes.
```

Attributes that are **domestic extensions** MUST be included in the **domestic namespaces** that are defined in Section :ref:`credential-data-model-pid:PID Data Model in SD-JWT-VC Format` and Section :ref:`credential-data-model-pid:PID Data Model in mdoc-CBOR Format` for SD-JWT-VC and mdoc-CBOR PIDs respectively.

PID Data Model in SD-JWT-VC Format
-----------------------------------

> For the SD-JWT-VC PID defined in this specification, the ``vct`` value MUST be ``urn:eudi:pid:it:1`` in compliance with ARF PID Rulebook v1.3 requirements for domestic PID extensions (requirement **PID_14**, Section 4.2, extending the base type ``urn:eudi:pid:``).

Reviewer: ```suggestion
The SD-JWT-VC PID defined in this specification MUST use the ``vct`` claim value set with ``urn:eudi:pid:it:1``, according to the domestic PID extensions defined in the ARF PID Rulebook v1.3 (see also ARF HLR **PID_14**, Section 4.2, extending the base type ``urn:eudi:pid:``).
```

 * Contents:
   ``["YWtI06xDdCyvTalcInTE3A", "birthdate", "1980-01-10"]``

**Claim** ``tax_id_code``:

 * SHA-256 Hash: ``_C7hoKFt0kV190v2GXIwLUIiDbc_7LcyofQmgDfute8``
 * Disclosure:
   ``WyItejM0Y0oxZ0M1VUJQQ0l4OE9oTmlRIiwgInRheF9pZF9jb2RlIiwgIlRJ``
   ``TklULVhYWFhYWFhYWFhYWFhYWFgiXQ``
 * Contents:
   ``["-z34cJ1gC5UBPCIx8OhNiQ", "tax_id_code",``
   ``"TINIT-XXXXXXXXXXXXXXXX"]``

**Claim** ``place_of_birth``:

 * SHA-256 Hash: ``tI5s2A_Ez6oZv6plZzUPjYAL-SJGiAUFyRbhzLsluGU``
 * Disclosure:
   ``WyJYY1hsUFZDcWpITnZlQkNubFZQWWdBIiwgInBsYWNlX29mX2JpcnRoIiwg``
   ``eyJsb2NhbGl0eSI6ICJSb21hIn1d``
 * Contents:
   ``["XcXlPVCqjHNveBCnlVPYgA", "place_of_birth", {"locality":``
   ``"Roma"}]``

**Claim** ``nationalities``:

 * SHA-256 Hash: ``GHYjuGUthjtB4q4Oz_ZSGPmCokLOpv2kpFNzz1LfFUY``
 * Disclosure:
   ``WyJLTmM1LUdrOUNRaF9UZEdicUJLSTdBIiwgIm5hdGlvbmFsaXRpZXMiLCBb``
   ``IklUIl1d``
 * Contents:
   ``["KNc5-Gk9CQh_TdGbqBKI7A", "nationalities", ["IT"]]``

The combined format for the PID issuance is given by:

.. literalinclude:: ../../examples/pid-sd-jwt-example-combined.txt
  :language: text

PID Data Model in mdoc-CBOR Format
----------------------------------------

> The PID in mdoc-CBOR format MUST use the **docType** ``eu.europa.ec.eudi.pid.1`` in compliance with ARF requirement **PID_04**.

Reviewer: ```suggestion
The PID in mdoc-CBOR format MUST use the **docType** ``eu.europa.ec.eudi.pid.1`` in compliance with ARF HLR **PID_04**.
```

we shoudl add in the acronyms ARF HLR as well
